StyleGuide.md

```diff
@@ -821,6 +821,16 @@ Highlight shell and SQL commands where appropriate using the following info:
 
 Start shell code samples with `~~~ shell` followed by a line break. The first character of the next line must be the terminal marker `$`. For multi-line shell commands, use a backslash (`\`) at the end of each line to indicate a line break.
 
+In an example command to download an artifact using the command line, `curl` is preferred over `wget` because it is included in Linux, macOS, and recent builds of Windows. Use syntax like:
+
+```
+curl -o {optional_output_path}/{output_file_name} [-H {header_keys_and_values}] {url} 
```

Comment:
By (Linux man page) convention, options in square brackets are optional and I tried to use that convention here. I wanted to give some hints for writers who may be more used to `wget` than `curl`.

- I don't consider `-o` optional because leaving it off outputs the stream to standard output (including if it's a binary), for a very ugly experience, so I didn't surround it with square brackets. `wget` has a behavior difference here: By default, a binary is always downloaded, and the download has the same file name as the file portion of the download URL.
- I added the `{optional_output_path}` part to the `-o` argument to show how to specify a local download path with `curl` if you are used to passing `-P` to `wget` to designate a path. I was trying to show that you don't need an extra argument to specify the path in `curl`, just reference it.
- We have some CC API examples where explicitly pass in headers, so I wanted to give a clue about how you would do that with `curl`'s `-H` argument, but I didn't want to imply that the header is always required.
- I don't consider `{url}` to be optional as it shows what to download or what REST URL to hit.